app/components/search_status_component.rb

```diff
@@ -0,0 +1,47 @@
+class SearchStatusComponent < ApplicationComponent
+  attr_reader :developer
+
+  def initialize(developer)
+    @developer = developer
+  end
+
+  def render?
+    developer.actively_looking? || developer.open? || developer.not_interested?
```

Comment:
A few suggestions to reduce the logic on this line: 

```suggestion
    developer.search_status
```

or 

```suggestion
    developer.search_status.present?
```

or

```suggestion
    !developer.search_status.nil?
```

Reply:
Good call!